extensions/wikia/CommunityPage/templates/helpModule.mustache

```diff
@@ -2,5 +2,6 @@
 	<h2>{{helpModule.title}}</h2>
 	<a data-tracking="help-page-edit-page-link" href="{{helpModule.editPageLink}}">{{helpModule.editPage}}&nbsp;&rarr;</a><br/>
 	<a data-tracking="help-page-add-links-page-link" href="{{helpModule.addLinksPageLink}}">{{helpModule.addLinks}}&nbsp;&rarr;</a><br/>
-	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a>
+	<a data-tracking="help-page-add-new-page-link" href="{{helpModule.addNewPageLink}}">{{helpModule.addNewPage}}&nbsp;&rarr;</a><br/>
+	<a data-tracking="help-page-policy-link" href="{{helpModule.communityPolicyLink}}">{{helpModule.communityPolicy}}&nbsp;&rarr;</a>
```

Comment:
talked to Trevor, he'd like still to display the link. I asked him where it should point if empty but haven't got answer yet